extensions/lifecycle/tasks/LifecycleTask.js

```diff
@@ -134,13 +169,22 @@ class LifecycleTask extends BackbeatTask {
         async.waterfall([
             next => req.send((err, data) => {
                 if (err) {
+                    LifecycleMetrics.onS3Request(
+                        log,
+                        'ListObjects',
+                        'bucket',
+                        err.statusCode
```

Comment:
similarly, may be better to always pass err to `onS3Request` : and let the implementation decide to use statusCode field or otherwise.